srv/salt/ceph/tuned/mgr/default.sls

```diff
@@ -0,0 +1,19 @@
+/usr/lib/tuned/ses-mgr/tuned.conf:
+  file.managed:
+    - source: salt://ceph/tuned/ses-mgr/tuned.conf
+    - makedirs: True
+    - user: root
+    - group: root
+    - mode: 644
+
+apply tuned ses mgr:
+  cmd.run:
+    - name: "tuned-adm profile ses-mgr"
+
+start-tuned:
+  cmd.run:
```

Comment:
Instead of cmd.run for start and enable I believe it would be better to use the service command..

tuned:
  service.running:
    - enable: True